Description: Look in the correct directory for the application's icon
Author: Stephen Kitt <steve@sk2.org>
Last-Update: 2011-03-13

--- gdmap.orig/src/about.c
+++ gdmap/src/about.c
@@ -24,7 +24,7 @@
 #ifdef PACKAGE_SOURCE_DIR
     temp = g_strdup_printf("%s/data/%s", PACKAGE_SOURCE_DIR, filename);
 #else
-    temp = g_strdup_printf("%s/pixmaps/%s", PACKAGE_DATA_DIR, filename);
+    temp = g_strdup_printf("/usr/share/pixmaps/%s", filename);
 #endif
     pix = gdk_pixbuf_new_from_file(temp, NULL);
     g_free(temp);
--- gdmap.orig/src/gui_main.c
+++ gdmap/src/gui_main.c
@@ -829,7 +829,7 @@
 #ifdef PACKAGE_SOURCE_DIR
     temp = g_strdup_printf("%s/data/%s", PACKAGE_SOURCE_DIR, filename);
 #else
-    temp = g_strdup_printf("%s/pixmaps/%s", PACKAGE_DATA_DIR, filename);
+    temp = g_strdup_printf("/usr/share/pixmaps/%s", filename);
 #endif
     pix = gdk_pixbuf_new_from_file(temp, NULL);
     g_free(temp);
